dedicated-postgres-clusters.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Introducing Dedicated Clusters for PostgreSQL'
+description: 'We are announcing dedicated clusters, for more scalable and isolated PostgreSQL databases.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cover-dedicated-clusters.jpg
+  alt: Dedicated Clusters
+author: Simona Pencea, Matt Field
+date: 3-22-2024
+tags: ['engineering']
+published: true
+slug: postgres-dedicated-clusters
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/cover-dedicated-clusters.jpg
+---
+
+Today we are excited to announce dedicated clusters! Dedicated clusters provide better scalability, security and noisy neighbor isolation, as well as better economics at scale for your Postgres databases on Xata. We are rolling this out in an Early Access Program (EAP) and you can [request access starting today](https://5i8caik7lja.typeform.com/to/Gh5lin22).
+
+![Postgres branches allocated to shared and dedicated clusters](images/branches-clusters.png)
+
+The key features of dedicated clusters are:
+
+- Xata databases and branches can be allocated to a dedicated cluster. You can allocate a single branch (for example, only your production branch) or multiple branches (for example, all your development branches).
+- You'll be able to move Postgres databases between dedicated clusters with no or minimal downtime.
+- You can configure dedicated clusters with a configurable number of read replicas, and you can ask for them to be in different availability zones.
+- You can choose between a wide range of [instance types](https://xata.io/docs/dedicated-cluster#provisioned-instances) or you can define a minimum and maximum CPU/RAM configuration and let [auto-scaling](https://xata.io/docs/dedicated-cluster#auto-scaling-instances) choose the right size dynamically based on load. Scaling up is almost instant and scaling down is done automatically in a matter of minutes.
+- The per-GB cost of storage is significantly cheaper (20 times less!) than the storage cost for shared clusters.
+- The dedicated clusters are free from limitations we impose on our shared clusters to protect from abuse: rate and concurrency limits as well as some limitations on the SQL commands you can execute. You can see a comparison table in the [docs](https://xata.io/docs/postgres#supported-statements).
```

Comment:
```suggestion
- A dedicated cluster is free from all limitations we impose on our shared clusters to protect from abuse: [rate and concurrency limits](/docs/rest-api/limits) as well as some limitations on the [SQL commands](/docs/postgres#supported-statements) you can execute.
```

Reply:
shortened the sentence a bit, added a link to rate limits and made links relative.